diff --git a/seva/modules/autoencoder.py b/seva/modules/autoencoder.py
--- a/seva/modules/autoencoder.py
+++ b/seva/modules/autoencoder.py
@@ -10,8 +10,9 @@ class AutoEncoder(nn.Module):
     def __init__(self, chunk_size: int | None = None):
         super().__init__()
         self.module = AutoencoderKL.from_pretrained(
-            "stabilityai/stable-diffusion-2-1-base",
-            subfolder="vae",
+            "stabilityai/sd-vae-ft-mse",
+            revision="31f26fdeee1355a5c34592e401dd41e45d25a493",
+            use_safetensors=True,
             force_download=False,
             low_cpu_mem_usage=False,
         )
